suite-native/biometrics/src/components/BiometricOverlay.tsx

```diff
@@ -1,23 +0,0 @@
-import React from 'react';
```

Comment:
Please can we just change this file and rename in next PR so we keep the diff?

Reply:
rename reverted in 335eb3d80